C:\Python313\python.exe: can't open file 'C:\\Users\\Gold System\\AppData\\Local\\Temp\\_MEI143162\\cenariostestespegasus\\CadastrosAdicionais\\CadastrosCenáriosJazigos\\CadastrosJazigosLote\\cadastrodejazigo5ºcenario.py': [Errno 2] No such file or directory
